packages/components/src/text/test/text.styles.native.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
It would be great to have a way to share those tests between native and web. Those tests should stay in sync moving forward, the proposed approach doesn't guarantee it.

@koke and @hypest - how do you tackle in other places?

Reply:
I’m not aware that it is a solved issue. A simple solution for this is just import the web file in the native one.

A mor comprehensive solution would be that tests run in both platforms by default, unless they have the web.js or .native.js extensions, but that seems like a bigger effort, since I’m not sure how many of the existing web tests would pass on native.